src/Files.App.OpenDialog/FilesOpenDialog.cpp

```diff
@@ -21,7 +21,10 @@ using std::endl;
 
 CComPtr<IFileOpenDialog> GetSystemDialog()
 {
-	HINSTANCE lib = CoLoadLibrary(L"C:\\Windows\\System32\\comdlg32.dll", false);
+	WCHAR comdlg32Path[MAX_PATH];
+	ExpandEnvironmentStrings(L"%SYSTEMDRIVE%\\Windows\\System32\\comdlg32.dll", comdlg32Path, MAX_PATH - 1);
```

Comment:
We want to specifically use Unicode version of APIs.
```suggestion
	ExpandEnvironmentStringsW(L"%SYSTEMDRIVE%\\Windows\\System32\\comdlg32.dll", comdlg32Path, MAX_PATH - 1);
```

Reply:
L165 also uses `ExpandEnvironmentStrings`. Should we use `ExpandEnvironmentStringsW` here as well?

Reply:
Yeah as long as we use L"" over _T"".

Reply:
Thanks!

Reply:
Done. And I changed `%SYSTEMDRIVE%\Windows` to `%WINDIR`.